install/installer/pkg/components/usage/stripe.go

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the MIT License. See License-MIT.txt in the project root for license information.
+
+package usage
+
+import (
+	"github.com/gitpod-io/gitpod/installer/pkg/config/v1/experimental"
+	corev1 "k8s.io/api/core/v1"
+	"path/filepath"
+)
+
+func getStripeConfig(cfg *experimental.Config) (corev1.Volume, corev1.VolumeMount, string, bool) {
```

Comment:
This exists because it's essential to have a single place that can give us all of - volume, mount and the path where it will be. Before, these were scattered around which also scattered the logic for deciding if to add the argument